torch/quantization/fx/convert.py

```diff
@@ -335,11 +335,13 @@ def node_arg_is_quantized(node_arg: Any) -> bool:
         else:
             return False
 
-    def is_output_quantized(node: Node, obj: QuantizeHandler, qconfig: QConfigAny, modules: Dict[str, torch.nn.Module]) -> bool:
+    def is_output_quantized(
+            node: Node, obj: QuantizeHandler, qconfig: QConfigAny,
+            modules: Dict[str, torch.nn.Module], is_reference=False) -> bool:
         """ Check if output node is quantized or not """
         assert modules is not None
         # by default the output for a quantizable node is expected to be quantized
-        quantized = True
+        quantized = obj.is_output_quantized(qconfig, is_reference)
```

Comment:
Is the comment now out of date? It looks like we check this for each quantize handler object.

Reply:
Oh OK, yeah will update